### PR TITLE
Adding 2 new APIs for VMSS: Redeploy and PerformMaintenance

### PR DESCRIPTION
**Pull request by pratnala**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json [hunk 1]
```diff
@@ -2689,6 +2689,110 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/redeploy": {
+      "post": {
```
Comment (dsgouda) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: Please provide `x-ms-examples` for each of these operations.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please add examples for the operations

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json (hunk 1 above)
Comment (pratnala) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: This is just another standard operation. Similar ones already exist in the single VM case without examples.

Comment (dsgouda) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: I understand but we need examples to approve new changes, it helps us validate the examples with the operations described in the spec.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**